jms/artemis/src/test/resources/amq.yaml

```diff
@@ -0,0 +1,171 @@
+apiVersion: v1
```

Comment:
How was this yaml constructed? Was there a lot of trimming of the initial yaml?

Reply:
I have provided this file to Michal. OpenShift ships a template for AMQ 6.3, which is old (it's based on ActiveMQ 5, while AMQ 7 is based on ActiveMQ Artemis). I took the YAML file I had for the Thorntail messaging booster and edited it slightly.

Reply:
My thing I had used before Lada came to rescue was bananas, AMQ 6.3 from what I found in RHOAR examples.